### DISCUSSION TITLE
Allign to corner not working (with dual display) in 4.4.190

### DISCUSSION THREAD

**Discussion by Procstas** [Open-Shell/Open-Shell-Menu#1679] [General]
Allign custom button to corner of the screen is not working (with dual display) in OpenShell 4.4.190.
Button moves to corner on secondary display but stays centered on primary display.

**Comment by ge0rdi** [comment]
What Windows are you using?

Please, provide screenshots for better clarification (with and without setting enabled).

**Comment by Procstas** [comment]
Windows 11

 

When I try to take  a screencapture with the custom-button on the secondary display aligned to the corner, the button jumps back to the center, so I took a picture instead.
The custom-button on the primary screen always stays in the center.

**Reply by ge0rdi** [reply]
> Windows 11

What version of Windows 11? (Windows Settings -> About)

> so I took a picture instead.

Please, add your responses (including pictures) on Github web directly.
Replying to notification e-mail doesn't work well :(

> When I try to take  a screencapture with the custom-button on the secondary display aligned to the corner, the button jumps back to the center

So is it just some _temporary_ issue?
Like the button is at wrong position after some action (what?) and then is placed correctly (after some other action)?
It may be some _refreshing_ issue (that we don't handle some notifications properly).

Are you able to replicate it at will?
What do you need to do for it to behave in that weird way?

**Reply by Procstas** [reply]
Windows 11 Pro 22H2 22621.2134
Button jumps back to center position after various actions (screen capture, about)
Then can be positioned back to corner by switching to Aero and then back to Custom.
Seems to me a weird refresh issue of Open-Shell
Can be replicated at will.
Button on primary monitor always stays in center, whatever action or setting.

![2023-08-24_11-38-41](https://github.com/Open-Shell/Open-Shell-Menu/assets/35809749/939f2b9a-9ee5-4cb5-8a87-ef61d7d09493)
![2023-08-24_12-18-54a](https://github.com/Open-Shell/Open-Shell-Menu/assets/35809749/452382be-03eb-41a8-a45f-a292ea0ab973)

**Reply by ge0rdi** [reply]
> Then can be positioned back to corner by switching to Aero and then back to Custom.

Does it happen also without changing `Open-Shell` settings?
I mean during normal work with the computer.

**Comment by Procstas** [comment]
Jumping back to wrong (center) position whitout changing settings?  Yes

**Reply by ge0rdi** [reply]
Hmm, _center_ is not wrong position as you appear to be using _center aligned_ taskbar.
`Open-Shell` is meant to replace original Start button, so in your case it is supposed to appear in _center_.

The weird thing is when `Open-Shell` custom button is at different position than original Start button. As that is not intentional for sure.
This is what I'm asking, whether it appears during normal work (without any changes to `Open-Shell` settings).

**Comment by Procstas** [comment]
I now changed Windows 11 taskbar-setting from center aligned taskbar (default?) to left aligned taskbar, which solved the problem.
Thanks.
